website/docusaurus.config.js

```diff
@@ -70,7 +71,7 @@ export default async function createConfigAsync() {
             // Otherwise we risk losing the update on the next release.
             return `https://github.com/risc0/risc0/edit/main/website/api/${docPath}`;
           },
-          lastVersion: "1.0",
+          lastVersion: apiVersions[0],
```

Comment:
automatically maps to latest version, so that way we don't forget to update it

Reply:
I specifically pinned this to `1.0` since we already snapped `1.1` but we are pausing the final release. Once `1.1` is fully released, we can just remove `lastVersion` since the default already does this.